### ISSUE TITLE
Freezing PyQt5 window

### ISSUE THREAD

**Issue by cleric5** [Nuitka/Nuitka#1680]
Hi
My program runs in two thread, in loops. The first one launches the PyQt5 window, the second one promptly working and outputs data to the screen. When I run the code directly or build PyInstaller, everything works fine. And when I compile with Nuitka, the program works in the console, but it hangs in the PyQt5 window and only after the end of the program it outputs all the information. That is, it does not hang, but it turns out to be blocked.

I don't understand why it might be and how to fix it? Since I tried to compile mingw64 and msvc and with different options. But without result. Tell me at least in which direction to look. 

...
self.thread = QThread()
self.worker = Worker()
self.worker.moveToThread(self.thread)
...

**Comment by kayhayen** [comment]
You didn*t fill out the issue template, and do not provde a reproducer. This might be a duplicate of PyQt5 threading not working with Nuitka, for which the only solution is to use PySide2 or PySide6.

**Comment by cleric5** [comment]
> You didn*t fill out the issue template, and do not provde a reproducer. This might be a duplicate of PyQt5 threading not working with Nuitka, for which the only solution is to use PySide2 or PySide6.

I also thought about it, since there are no errors and it works with other systems. And how do I switch to PySide 2 when compiling Nuitka? I can't understand it. I tried to put python like this -m pip install -U --force-reinstall https://ghp_XXXXXXXX@github.com/Nuitka/Nuitka-commercial/releases/download/pyside2_5.15.2_wheels_v1/PySide2-5.15.2-5.15.2-cp27-cp27m-win_amd64.whl but it does not fit (I have windows 10 -64). 

How can I compile then under PySide 2?

**Comment by kayhayen** [comment]
There is more than one wheel needed, I think shiboken2 is also there. You should install them all at once.

**Comment by cleric5** [comment]
> There is more than one wheel needed, I think shiboken2 is also there. You should install them all at once.

After various tests, I see a version incompatibility error. I have python 3.10. And 3.9 is supported at most. So I can't install in your build and vice versa install the standard one from them?

python -m pip install -U --force-reinstall PySide-5.15.2-with-Nuitka.tar.gz

Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [1 lines of output]
      Unsupported python version detected. Only these python versions are supported: [(2, 7), (3, 5), (3, 6), (3, 7), (3, 8), (3, 9)]
      [end of output]
